app/server.js

```diff
@@ -282,8 +285,8 @@ export default function(req, res, next) {
     res.write('<head>\n');
 
     // Write preload hints before doing anything else
-    if (process.env.NODE_ENV !== 'development' && process.env.NODE_ENV !== 'production') {
-      if (config.GTMid) {
+    if (process.env.NODE_ENV !== 'development') {
+      if (!!config.GTMid) {
```

Comment:
Double negation???

Reply:
[that is transportkollektiv's modification](https://github.com/mfdz/digitransit-ui/pull/13/commits/fee0a929d3931593aff79b733e590f08453c4209), we didn't touch it

Reply:
it's an incoming change from your side, so it's something you also pulled from somewhere else or you made it yourself. Anyway, since the double negation has the same effect as the original if, please convert it to that because it's cleaner

Reply:
"!!" removed from server.js